src/foam/java/Method.js

```diff
@@ -30,12 +30,22 @@ foam.CLASS({
       of: 'foam.java.Argument',
       name: 'args'
     },
+    {
+      class: 'Boolean',
+      name: 'override',
+      value: false
+    },
     { class: 'StringArray', name: 'throws' },
     { class: 'foam.java.CodeProperty', name: 'body' }
   ],
 
   methods: [
     function outputJava(o) {
+      if (this.override) {
+        o.indent();
+        o.out("@Override\n");
+      }
```

Comment:
Tested again, extra indent is needed for proper formatting. Nice catch though :)